### PR TITLE
Add preferred windows installation instructions

### PR DESCRIPTION
**Pull request by matthewjberger**
This adds instructions on using [scoop](https://github.com/lukesampson/scoop) to install artifact.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ For Linux and Mac, simply download and unpack the tarball with
 [10]: http://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path
 
 ### Windows
+
+#### Scoop
+The recommended method of installation for windows is to use [scoop](https://github.com/lukesampson/scoop).
+First, install scoop by opening up a powershell terminal and running:
```
Comment (vitiral) [inline docs/Installation.md]: I would prefer just a web link to their installation instructions

File: docs/Installation.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ For Linux and Mac, simply download and unpack the tarball with
...
+
+then to install artifact:
+
+```
+scoop install artifact
+```
+
+#### Alternative Windows method
+
 > Note: windows guide is incomplete. Need to add how to add it to your "PATH"
```
Comment (vitiral) [inline docs/Installation.md]: remove this line

**Comment by matthewjberger** [comment]
I made the changes you requested. How does that look?

**Comment by vitiral** [comment]
Thanks! I pulled and made a few edits (I mostly went back you your original). Thanks for the PR :)

**Pull request closed without merging**